test/common/schemaform/helpers.unit.spec.js

```diff
@@ -741,6 +741,33 @@ describe('Schemaform helpers:', () => {
 
       expect(output.address).to.be.undefined;
     });
+    it('should remove empty objects', () => {
+      const formConfig = {
+        chapters: {
+          chapter1: {
+            pages: {
+              page1: {}
+            }
+          }
+        }
+      };
+      const formData = {
+        page1: {
+          data: {
+            someField: {
```

Comment:
If the form config is empty, does it matter whether these fields are empty, undefined, or defined? Just wondering what scenario this is testing as opposed to
```
formData = {};
```
or 
```
formData = {
  page1: {
    data: {}
  }
};